### PR TITLE
Updating broken example in readme

### PR DESCRIPTION
**Pull request by jamesrweb**
One of the examples in the readme is broken, I updated the docs to fix this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,16 @@ function sketch(p5) {
 }
 
 function App() {
+  const [rotation, setRotation] = useState(0);
+
+  useEffect(() => {
+    const interval = setInterval(rotation => setRotation(rotation + 1), 100);
```
Comment (jamesrweb) [inline README.md]: @yevdyko maybe this isn't a good example but it fixes the bug, we could replace this with something else, WDYT?

Comment (yevdyko) [reply README.md]: It's still broken:
- `useState` and `useEffect` must be imported from `React`. 
- `rotation` is `NaN` in props.

**Review by yevdyko — CHANGES_REQUESTED** [review]
@jamesrweb can you double-check the fix?

File: README.md (hunk 1 above)
Comment (jamesrweb) [reply README.md]: Updated the example and tested locally 👍🏻

**Comment by jamesrweb** [comment]
@yevdyko can you re-read and re-test please, thanks! 🙏🏻

**Review by yevdyko — APPROVED** [review]
Looks awesome! Thanks @jamesrweb for the fix!

🚢

**Comment by jamesrweb** [comment]
> Looks awesome! Thanks @jamesrweb for the fix!
> 
> 
> 
> 🚢 

🥳

**Pull request merged**